### ISSUE TITLE
LinearAlgebra/addmul:192 rtol error on CI

### ISSUE THREAD

**Issue by vtjnash** [JuliaLang/julia#49966]
```
The global RNG seed was 0x65b5a36e0dc8b90ff93bf7e957e6c9ea.
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[0.007323434280788799 + 0.16639231153911094im 0.19182176800035539 - 0.1506834249491118im; -0.33937619573445627 + 0.6307676393543213im -0.21001740927092938 + 0.24231629124409637im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[0.7692130991106799 - 1.0129807818729184im -0.875095711849474 + 0.8113900526267961im; -0.055415854197585115 - 0.5040038767901225im 0.7194807004525097 - 0.6578745906003649im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[7.68477709782902 + 4.295945787832917im -5.9118145257868395 - 1.397838171515129im; 2.5044458382667782 - 2.954527954366954im -2.491757604732268 - 6.188371473838933im] ≈ ComplexF64[3.9830977675878927 + 3.6016394809146037im -6.591472895707831 - 2.254652426444485im; 1.7896816750291233 - 3.5761699422715783im -0.6045615932293433 - 3.11201101614523im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[15.064212666055166 + 4.207395261806841im -8.08086047344678 + 0.6675660561871892im; 3.5090654830258905 - 4.073139881933784im -5.266180182827892 - 11.44122326613291im] ≈ ComplexF64[11.888089409845435 + 3.9092468595804384im -7.161667767892271 - 1.9213334036627812im; 0.6616629316139058 - 1.1100406886670786im -4.213014368350767 - 8.064149860920057im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[24.653490473988107 + 3.0709639020202637im -10.61722509343417 + 3.6550723682534185im; 4.661395180172715 - 5.667614458213355im -9.602962559355184 - 17.89187090954099im] ≈ ComplexF64[21.477367217778376 + 2.772815499793861im -9.69803238787966 + 1.0661729084034484im; 1.81399262876073 - 2.7045152649466493im -8.54979674487806 - 14.514797504328136im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[0.7692130991106799 - 1.0129807818729184im -0.875095711849474 + 0.8113900526267961im; -0.055415854197585115 - 0.5040038767901225im 0.7194807004525097 - 0.6578745906003649im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[0.7692130991106799 - 1.0129807818729184im -0.875095711849474 + 0.8113900526267961im; -0.055415854197585115 - 0.5040038767901225im 0.7194807004525097 - 0.6578745906003649im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.470892429351807 - 0.3186744749546051im -0.19543734192848206 + 1.6682043075561523im; 0.6593483090400696 + 0.11763811111450195im -1.167715311050415 - 3.7342350482940674im] ≈ ComplexF64[1.2947691731420756 - 0.6168228771810076im 0.7237553636260269 - 0.9206951522938179im; -2.188054242371915 + 3.0807373043812074im -0.11454949657329051 - 0.357161643081215im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[5.490121856615694 - 0.1378720378136854im 1.1320426482887085 + 1.282812085434092im; 0.6148195009853483 + 1.3210857889686254im -1.2300462102511833 - 3.769017309070151im] ≈ ComplexF64[1.7884425263745674 - 0.8321783447319987im 0.45238427836771655 + 0.4259978305047359im; -0.09994466225230643 + 0.699443801064001im 0.6571498012517414 - 0.6926568513764484im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[5.3687642975249865 - 4.065585573793422im 0.8786532333301347 + 1.1323737194580505im; 1.665992357233127 - 0.06063118890285127im -3.9450194686758415 - 3.580892987234309im] ≈ ComplexF64[2.1926410413152553 - 4.363733976019825im 1.7978459388846435 - 1.4565257403919198im; -1.1814101941788577 + 2.902468004363854im -2.891853654198717 - 0.20381958202145667im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[2.684066401386389 - 4.69253199954922im 0.7309097880793984 + 1.2769941059264545im; 0.9190689190740848 - 1.022765735374815im -4.307719081457145 - 1.7063136781780877im] ≈ ComplexF64[-0.49205685482334194 - 4.990680401775623im 1.6501024936339075 - 1.311905353923516im; -1.9283336323379 + 1.9403334578918905im -3.2545532669800203 + 1.6707597270347647im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[3.5241611909276562 + 1.6372607044435927im -0.8492672347385248 + 1.18285091680177im; 0.4505399897085494 + 0.36629741449633935im 0.68385783496468 - 3.3223730666703526im] ≈ ComplexF64[1.0089961274913162 - 0.4448695419056947im -1.0045667196158778 + 0.24750757440250903im; 0.1697881077428851 - 0.40547027147240905im 0.8220498444466692 - 0.1964959644078466im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[3.5241611909276562 + 1.6372607044435927im -0.8492672347385248 + 1.18285091680177im; 0.4505399897085494 + 0.36629741449633935im 0.68385783496468 - 3.3223730666703526im] ≈ ComplexF64[1.2410337340074806 + 0.07640023150995573im 0.93574257128234 - 0.39399396499307776im; -2.9538726042923162 + 1.416557413567487im 0.06323662605732053 - 0.3189722666819018im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[3.5241611909276562 + 1.6372607044435927im -0.8492672347385248 + 1.18285091680177im; 0.4505399897085494 + 0.36629741449633935im 0.68385783496468 - 3.3223730666703526im] ≈ ComplexF64[1.2410337340074806 + 0.07640023150995573im 0.93574257128234 - 0.39399396499307776im; -2.9538726042923162 + 1.416557413567487im 0.06323662605732053 - 0.3189722666819018im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[3.5241611909276562 + 1.6372607044435927im -0.8492672347385248 + 1.18285091680177im; 0.4505399897085494 + 0.36629741449633935im 0.68385783496468 - 3.3223730666703526im] ≈ ComplexF64[1.0089961274913162 - 0.4448695419056947im -1.0045667196158778 + 0.24750757440250903im; 0.1697881077428851 - 0.40547027147240905im 0.8220498444466692 - 0.1964959644078466im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[3.5241611909276562 + 1.6372607044435927im -0.8492672347385248 + 1.18285091680177im; 0.4505399897085494 + 0.36629741449633935im 0.68385783496468 - 3.3223730666703526im] ≈ ComplexF64[1.2410337340074806 + 0.07640023150995573im 0.93574257128234 - 0.39399396499307776im; -2.9538726042923162 + 1.416557413567487im 0.06323662605732053 - 0.3189722666819018im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[3.5241611909276562 + 1.6372607044435927im -0.8492672347385248 + 1.18285091680177im; 0.4505399897085494 + 0.36629741449633935im 0.68385783496468 - 3.3223730666703526im] ≈ ComplexF64[1.2410337340074806 + 0.07640023150995573im 0.93574257128234 - 0.39399396499307776im; -2.9538726042923162 + 1.416557413567487im 0.06323662605732053 - 0.3189722666819018im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[4.310492545619812 + 2.251376717072933im 0.18071195753577496 + 1.2626419742763204im; 0.06371932399964081 + 1.2427258493894966im 0.6481956706724694 - 3.4873826028503143im] ≈ ComplexF64[1.7953274821834726 + 0.16924647072364574im 0.025412472658421947 + 0.3272986318770595im; -0.2170325579660235 + 0.4709581634207481im 0.7863876801544585 - 0.3615055005878082im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[5.828815240239603 - 0.8785007945197276im 0.03936890447456021 + 1.288330751203631im; 1.3045488772014684 + 0.547495921437416im -1.5633617846173207 - 4.391526573092305im] ≈ ComplexF64[3.545687783319427 - 2.4393612674533647im 1.824378710495425 - 0.28851413059121667im; -2.099863716799397 + 1.5977559205085636im -2.18398299352468 - 1.3881257731038543im] (rtol=0.0003452669770922512)
 
Error in testset LinearAlgebra/addmul:
Test Failed at /cache/build/default-amdci4-7/julialang/julia-master/julia-196ce53c6d/share/julia/stdlib/v1.10/LinearAlgebra/test/addmul.jl:192
Expression: ≈(collect(returned_mat), α * Ac * Bc + β * Cc, rtol = rtol)
Evaluated: ComplexF64[3.9810827416274503 - 2.473000879967956im 0.03126247992859055 + 1.388792535026057im; 1.0572737777148635 - 0.4192581651500118im -2.5756613026906976 - 3.0554658929431393im] ≈ ComplexF64[1.6979552847072747 - 4.033861352901593im 1.8162722859494553 - 0.18805234676879065im; -2.347138816286002 + 0.6310018339211358im -3.196282511598057 - 0.05206509295468853im] (rtol=0.0003452669770922512)
```
https://buildkite.com/julialang/julia-master/builds/24304#018858f7-aed5-4691-852a-b9933dabbbc6

**Comment by LilithHafner** [comment]
Is there a merged PR for this or is it tracked elsewhere?

**Comment by ViralBShah** [comment]
I just closed it since we don't see this anywhere. Would it be better to leave it open?

**Comment by LilithHafner** [comment]
This issue no longer makes CI red because the test is skipped (https://github.com/JuliaLang/julia/commit/13e40afdec396211efaeadf641cd0b37322ae378) but the problem has yet to be diagnosed and fixed.

**Comment by giordano** [comment]
I can't reproduce this failure:
```julia-repl
julia> Base.runtests("LinearAlgebra/addmul"; seed=0x65b5a36e0dc8b90ff93bf7e957e6c9ea)
Running parallel tests with:
  getpid() = 3528728
  nworkers() = 1
  nthreads() = 1
  Sys.CPU_THREADS = 36
  Sys.total_memory() = 572.056 GiB
  Sys.free_memory() = 516.516 GiB

Test             (Worker) | Time (s) | GC (s) | GC % | Alloc (MB) | RSS (MB)
LinearAlgebra/addmul  (1) |        started at 2024-09-14T00:06:12.957
LinearAlgebra/addmul  (1) |   114.96 |   2.73 |  2.4 |   16874.10 |  1518.50

Test Summary: | Pass  Broken  Total     Time
  Overall     | 4392    2862   7254  1m56.3s
    SUCCESS

julia> versioninfo()
Julia Version 1.12.0-DEV.1203
Commit 2ee655139d* (2024-09-13 22:47 UTC)
Platform Info:
  OS: Linux (aarch64-redhat-linux)
  CPU: 72 × unknown
  WORD_SIZE: 64
  LLVM: libLLVM-18.1.7 (ORCJIT, neoverse-v2)
Threads: 1 default, 0 interactive, 1 GC (on 72 virtual cores)
```

**Comment by LilithHafner** [comment]
You'll need to revert 13e40afdec396211efaeadf641cd0b37322ae378 to have a short at reproducing this failure.

**Comment by giordano** [comment]
I still can't reproduce a failure with
```diff
diff --git a/stdlib/LinearAlgebra/test/addmul.jl b/stdlib/LinearAlgebra/test/addmul.jl
index 3fff828924..413db9a4d3 100644
--- a/stdlib/LinearAlgebra/test/addmul.jl
+++ b/stdlib/LinearAlgebra/test/addmul.jl
@@ -165,7 +165,7 @@ end
         returned_mat = mul!(C, A, B, α, β)
         @test returned_mat === C
         # This test is skipped because it is flakey, but should be fixed and put back (see #49966)
-        @test_skip collect(returned_mat) ≈ α * Ac * Bc + β * Cc  rtol=rtol
+        @test collect(returned_mat) ≈ α * Ac * Bc + β * Cc  rtol=rtol
 
         y = C[:, 1]
         x = B[:, 1]
@@ -191,7 +191,7 @@ end
                     returned_mat = mul!(C, Af, Bf, α, β)
                     @test returned_mat === C
                     # This test is skipped because it is flakey, but should be fixed and put back (see #49966)
-                    @test_skip collect(returned_mat) ≈ α * Ac * Bc + β * Cc  rtol=rtol
+                    @test collect(returned_mat) ≈ α * Ac * Bc + β * Cc  rtol=rtol
                 end
             end
         end
@@ -204,7 +204,7 @@ end
                 returned_mat = mul!(C, A, B, α, zero(eltype(C)))
                 @test returned_mat === C
                 # This test is skipped because it is flakey, but should be fixed and put back (see #49966)
-                @test_skip collect(returned_mat) ≈ α * Ac * Bc  rtol=rtol
+                @test collect(returned_mat) ≈ α * Ac * Bc  rtol=rtol
             end
         end
 
```
but who knows if the seed is still good to reproduce the original issue (for example if other tests introduced in the meantime affect the RNG state)

**Comment by LilithHafner** [comment]
If we can't reproduce the failure, maybe it's time to put those tests back (#55775). Though it's suspicious that AFAICT nobody identified or intentionally fixed the bug that was causing these failures.
